Snippets/WebsocketTest.swift

```diff
@@ -13,11 +14,21 @@ router.get { _, _ in
 }
 
 router.ws("/ws") { inbound, outbound, _ in
-    for try await packet in inbound {
-        if case .text("disconnect") = packet {
+    for try await frame in inbound {
+        if frame.opcode == .text, String(buffer: frame.data) == "disconnect", frame.fin == true {
```

Comment:
I like this a _lot_, since we can now also decode a `frame.opcode == .text` into JSON without having to do the dance of converting to String inbetween. Because currently, we'd need to convert `ByteBuffer` into `String`, then into `Data` again.

Reply:
It is like this because a single frame fragment can have incomplete UTF8 data, so we can't decode it until we have all the frames that make up a message.

Unfortunately the WebSocketMessage is a String. Should I consider changing that back to a ByteBuffer.